dev/dll/Microsoft.UI.Xaml.vcxproj

```diff
@@ -72,14 +72,12 @@
   <ItemGroup>
     <Midl Include="..\..\idl\Microsoft.UI.Xaml.idl" />
   </ItemGroup>
-
```

Comment:
Just a stylistic note, but I'd actually prefer it if you reverted the changes to this file, then reapplied the salient parts (the change in version number in the `<Import />` paths) by hand. The Visual C++ project system has a bad habit of destroying blank lines and comments whenever it updates a project file. This would also make this file much easier to diff. Thanks!

Reply:
That's fair. The problem is it's just going to happen to the next person that uses VS to modify the project file and I'd rather just get it done. I could try to separate it into a separate PR though.